### ISSUE TITLE
Chrome warning about changing new tab blinks for a moment every time.

### ISSUE THREAD

**Issue by artquest** [jimschubert/NewTab-Redirect#171]
Hello,
on one of my virtual machines (Win 10 x64) every time I start Chrome a blinking warning "Is this the new tab page you were expecting?" appears for a moment so I don't have time to press "Keep changes".
Is it a bug or the way it should be?

![Snap13](https://user-images.githubusercontent.com/24230703/69538855-4aa31680-0f94-11ea-8dde-ade0812cda8d.png)

**Comment by jimschubert** [comment]
@artquest I don't think this would be expected. However, it's controlled by the browser and not by extensions. I _believe_ you can acknowledge this in chrome://settings as well. You'll usually see a message like this if there are more than one override extension installed. I'd recommend seeing if you have vm browser extensions (Parallels, for instance) with New Tab override permissions, and disabling that or New Tab Redirect. You could also open a Chrome bug for a longer term but slower resolution.

**Comment by artquest** [comment]
Found the solution: I changed the redirect URL to about:blank and next time I start Chrome the warning didn't disappear.
